topics/statistics/tutorials/aberrant_pi3k_pathway_analysis/tutorial.md

```diff
@@ -0,0 +1,643 @@
+>    - Log file for script run and additional information
+>    - list of classifier_summary.txt for each disease
+>    - list of classifier_coefficients.tsv for each disease
+>    - list of pancan_roc_results.tsv for each disease
+>    - list of summary_counts.csv for each disease
+>    - Within disease figures:  List of 5 files [all_disease_pr, all_disease_roc, cv_heatmap, disease_pr, disease_roc] for individual diseases
+>    - Disease classifier figures: list of 2 files [disease_pr disease_roc] for each disease
+{: .comment}
+
+## **PanCancer compare within models**
+we next do a performance comparison between the ERBB2,PIK3CA,KRAS,AKT1 pan model and individual models.
+![Compare Pan model with with-in disease model](../../images/aberrant_pi3k_pathway_analysis/within.png "Cross-validation performance characteristic metric AUROC for the pan-cancer model compared to individual models trained on each cancer type")
+
+> ### {% icon hands_on %} Hands-on: compare the ERBB2_PIK3CA_KRAS_AKT1 pan model with individual disease models
+>
+> 1. {% tool [PAPAA: PanCancer compare within models](toolshed.g2.bx.psu.edu/repos/vijay/pancancer_compare_within_models/pancancer_compare_within_models/0.1.9) %} with the following parameters:
+>    - {% icon param-file %} *"pancancer classifier summary"*: `classifier_summary.txt` (output of **PAPAA: PanCancer classifier** {% icon tool %})
+>    - {% icon param-file %} *"pancancer classifier coefficients"*: `classifier_coefficients.tsv` (output of **PAPAA: Pancancer classifier** {% icon tool %}) 
+>    - {% icon param-file %} *"pan_within classifier summary"*: `classifier_summary` (output of **PAPAA: PanCancer within disease analysis** {% icon tool %})(Note: Select all pan-within classifier summaries as dataset collection instead of multiple datasets)
+>    - {% icon param-file %} *"pan_within classifier coefficients"*: `classifier_coefficients` (output of **PAPAA: Pancancer within disease analysis** {% icon tool %}) (Note: Select all pan-within classifier coefficients as dataset collection instead of multiple datasets)
+>    - *"Would you want to compare given model with alt gene model?"*: `do not do alt gene`
+{: .hands_on}
+
+> ### {% icon details %} PanCancer Aberrant Pathway Activity Analysis `compare_within_models.R` inputs
+>    - \--pancan_model: location of pan and pan-within-disease models classifier_summary.txt/s and classifier_coefficients.tsv/s dataset collections.
+>    - \--alt_model: location of alt and alt-within-disease models classifier_summary.txt/s and classifier_coefficients.tsv/s dataset collections.
+{: .details}
+
+> ### {% icon comment %} Outputs
+>    - Comparison plots for Pan and Pan_within models ("auroc_comparison.pdf" and "aupr_comparison.pdf")
+>    - Comparison plots for altgene, alt_within, Pan_alt models ("alt_gene_auroc_comparison.pdf" and "alt_gene_aupr_comparison.pdf")
+{: .comment}
+
+## **PanCancer apply weights**
```

Comment:
@shiltemann appreciate your suggestion, and the tools are in the order of processing the data in a step by step manner. Right from building the classifier, applying weights, additional steps, visualization, external data evaluation, and pharmacological data processing. I will take a look at the generated agenda and will see if I can improve on that.

Reply:
yes that is great, and you already name some nice section names there! just group tools/tools into those sections and you're pretty much there :+1:

Reply:
did you get a chance to look at this? perhaps some of these tools can be grouped together under a conceptual step, rather than using the tool names as section headers?

e.g. modelling, visualization, pathway analysis?  (I am not familiar with the topic, so up to you, but could be nice as an extra guide to understand the different parts of the workflow)

![image](https://user-images.githubusercontent.com/2563865/109955611-f7fdf180-7ce2-11eb-8d99-02529640ab01.png)